stories/LocationSelector.tsx

```diff
@@ -41,6 +41,7 @@ storiesOf('Organisms|LocationSelector', module)
 
         return (
             <LocationSelector
+                className="test-class"
```

Comment:
Maybe add a test?

Reply:
what sort of test do you have in mind?

Reply:
A test that ensures custom html attributes can be applied (without causing errors, but I guess that's already covered with linting) like you seem to have added this line for